device/core/src/interfaces.ts

```diff
@@ -5,6 +5,7 @@
 
 import { X509 } from 'azure-iot-common';
 import { Agent } from 'https';
+import { HttpOperationResponse } from '@azure/core-http';
```

Comment:
>@azure/core-http [](start = 39, length = 16)

that seems like a big dependency to take on for just a type

Reply:
I will stub the HttpOperationResponse